mm2src/coins/z_coin/z_rpc.rs

```diff
@@ -227,6 +231,19 @@ impl ZRpcOps for LightRpcClient {
             sapling_tree,
         }))
     }
+
+    #[cfg(not(target_arch = "wasm32"))]
+    async fn get_lightd_info(&mut self) -> MmResult<LightdInfo, UpdateBlocksCacheErr> {
```

Comment:
This is not used anywhere, I understand that you added it to get the `sapling_activation_height`. You can either use it or use the one in the config to calculate if the date/height requested is pre-sapling or not.

Reply:
I'll go with this instead

Reply:
Please note that [`librustzcash`](https://github.com/KomodoPlatform/librustzcash/blob/636845d0f93d9b797ed9686ae43d7abff11b1783/zcash_client_backend/src/data_api/chain.rs#L144-L145) use the one from config https://github.com/KomodoPlatform/komodo-defi-framework/blob/2d4eb0e81ed4d979824b6cc06d1cd75e9b771dad/mm2src/coins/z_coin.rs#L173
It's good to use the same source of information everywhere, using config will also reduce the calls by one.